docker-compose.yml

```diff
@@ -107,18 +127,35 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
       - POSTGRES_DB=tecken
+    healthcheck:
+      test: ["CMD", "pg_isready"]
+      interval: 10s
```

Comment:
Maybe we want something shorter here? `pg_isready` returns immediately if the DB isn't up yet, so we'd wait for 10 seconds if it isn't up the first time the health check is run, which can be annoyingly long in interactive usage. If you are worried that a shorter interval isn't long enough, you can always increase the number of retries. Since these checks are all quite cheap, we can afford running them a few times.

(Same for some of the other intervals further down.)

Reply:
good point, I changed the interval of all of these to 1s and timeout 3s and that seems to work fine for me locally